modules/styling/lib/cs.ts

```diff
@@ -445,10 +445,10 @@ export type CSToPropsInput =
   | undefined
   | CS
   | CsToPropsReturn
-  | Properties<string | number>
+  | CSSObjectWithVars
   | CSToPropsInput[];
 
-export type CsToPropsReturn = {className?: string; style?: Properties<string | number>};
+export type CsToPropsReturn = {className?: string; style?: Record<string, string>};
```

Comment:
Stencils return `Record<string, string>` and nothing else uses `CsPropsReturn` externally

Reply:
Seems like this line cause type check failure. Can it be `CSSObjectWithVars` instead of `Record<string, string>`? Using `CSSObjectWithVars` removing this issue